rapids-cmake/cpm/nvcomp.cmake

```diff
@@ -143,10 +143,10 @@ function(rapids_cpm_nvcomp)
 
       # Record the nvcomp_DIR so that if USE_PROPRIETARY_BINARY is disabled we can safely clear the
       # nvcomp_DIR value
-      set(nvcomp_proprietary_binary_dir "${nvcomp_ROOT}/${lib_dir}/cmake/nvcomp")
-      cmake_path(NORMAL_PATH nvcomp_proprietary_binary_dir)
-      set(rapids_cpm_nvcomp_proprietary_binary_dir "${nvcomp_proprietary_binary_dir}"
-          CACHE INTERNAL "nvcomp proprietary location")
+      set(nvcomp_proprietary_root "${nvcomp_ROOT}")
+      cmake_path(NORMAL_PATH nvcomp_proprietary_root)
+      set(rapids_cpm_nvcomp_proprietary_root "${nvcomp_proprietary_root}"
+          CACHE INTERNAL "nvcomp proprietary root dir location")
```

Comment:
How will this affect the `cpm_nvcomp-proprietary_binary-lib-location-nested` test that was added in #675? It seems to me that it will no longer find the file if it's in `nested/lib64/cmake/nvcomp`, since that's not one of the places `find_package()` looks. Requesting clarification here.

Reply:
The `nvcomp_proprietary_binary_dir` was never used for searches so it shouldn't have any imact.
The `nvcomp_DIR` cache variable will hold the found location from the first execution and be used for all following executions.